Funcky/Funcky.csproj

```diff
@@ -27,6 +27,9 @@
         <AdditionalFiles Include="PublicAPI.Shipped.txt" />
         <AdditionalFiles Include="PublicAPI.Unshipped.txt" />
     </ItemGroup>
+    <ItemGroup>
+      <Folder Include="DataTypes\" />
```

Comment:
I don't think this is needed anymore

Reply:
Fixed 4039799298d074f0a3e01d65408692ce214c1164